package.json

```diff
@@ -144,9 +148,21 @@
       "plugin:@typescript-eslint/recommended",
       "plugin:@typescript-eslint/recommended-requiring-type-checking",
       "plugin:import/recommended",
-      "plugin:import/typescript"
+      "plugin:import/typescript",
+      "prettier"
     ]
   },
+  "lint-staged": {
+    "src/**/*.{ts,tsx,js}": "prettier -w --ignore-unknown"
+  },
+  "prettier": {
+    "parser": "typescript",
+    "singleQuote": true,
+    "jsxSingleQuote": true,
+    "semi": false,
+    "tabWidth": 2,
+    "trailingComma": "none"
```

Comment:
This seems to remove many commas we wrote, maybe this option should not be `none`?

Reply:
If you check our `.editorconfig` then you'll see that we set `ij_typescript_enforce_trailing_comma = remove`. So I guess we always wanted to remove the commas.

Reply:
Then I withdraw my initial comment…